src/EPR.Accreditation.Facade.Common/RESTservices/Interfaces/IHttpSiteService.cs

```diff
@@ -8,7 +9,10 @@ public interface IHttpSiteService
            Guid siteExternalId);
    
         Task UpdateSite(
-            Guid accreditationExternalId,
+            Guid siteExternalId,
```

Comment:
This should not be the site id, this should be the accreditation id - there is a one to one relationship to site from accreditation, so you never need the site id (unless it is an overseas site, which i do not think this work is for)

Reply:
Done.